specs/commandline/plugin.md

```diff
@@ -36,15 +38,53 @@ Aliases:
   list, ls
 ```
 
+### notation plugin install
+
+```text
+Installs a plugin
+
+Usage:
+  notation plugin install [flags] <plugin path>
+
+Flags:
+  -h, --help                    help for install
+
+Aliases:
+  install, add
+```
+
+### notation plugin remove 
+
+```text
+Removes a plugin
+
+Usage:
+  notation plugin remove [flags] <plugin name>
+
+Flags:
+  -h, --help          help for remove
+
+Aliases:
+  remove, rm, uninstall, delete
+```
+
 ## Usage
 
 ### Install a plugin
 
-Currently there is no subcommand available for plugin installation. Plugin publisher should provide instructions to download and install the plugin.
+```shell
+notation plugin install <plugin path>
+```
+
+Upon successful execution, the plugin is moved to plugins directory. If the plugin directory does not exist, it will be created. The plugin installation is then verified and if the verification fails, the plugin is removed from the plugins directory and an error is returned. When an existing plugin is detected, the versions are compared and if the existing plugin is a lower version then it is replaced by the newer version.
```

Comment:
```suggestion
Upon successful execution, the plugin is moved to plugins directory and name+version of plugin is displayed. If the plugin directory does not exist, it will be created. When an existing plugin is detected, the versions are compared and if the existing plugin is a lower version then it is replaced by the newer version.
```

How can customer lower the version, probably we should add --force command ?

Reply:
Thank you for the feedback and suggestion, I updated the install description and added a `--force` flag to the spec. :)